types/site/_data/authorsData.d.ts

```diff
@@ -22,6 +22,8 @@ declare global {
     homepage?: string;
     image?: string;
     twitter?: string;
+    ddc?: string;
```

Comment:
The internal abbreviation we use is `dcc`, this needs to be updated in a bunch of places in this PR.